src/components/Book.css

```diff
@@ -1,11 +1,13 @@
 .bookSection {
-  width: 80%;
+  width: 79%;
   height: 9rem;
-  margin: 2.313rem 6.25rem 0;
-  padding: -4rem 6.188rem 1.625rem 1.688rem;
-  border-radius: 4px;
+  padding: -4rem 6.17rem 1.625rem 6.25rem;
+  border-radius: 6px;
   border: solid 1px #e8e8e8;
   background-color: #fff;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 1rem;
 }
 
 .bookContainer {
```

Comment:
- [ ] Kindly make sure each bookContainer block sticks as much as possible the template(e.g.: fonts, size, spacing,...) design for the project requirements.
### Template design
![Screenshot from 2022-11-11 18-08-07](https://user-images.githubusercontent.com/83042742/201383276-d3b9fe15-b9e8-4296-bb0c-8d2f66520461.png)

### Your design
![Screenshot from 2022-11-11 18-10-48](https://user-images.githubusercontent.com/83042742/201383329-b8535ab9-57e0-4c32-b116-e2912ad94757.png)